dashboard/app/views/videos/embed.html.haml

```diff
@@ -1,6 +1,14 @@
+/ This view creates a page that transparently provides fallback-player support to the specified YouTube video.
+/ See VideosController#embed for usage
+!!! 5
+- video = local_assigns[:video]
+- process_fallback_videos = current_user.try(:admin?) && !(Rails.env.production? || Rails.env.test?)
```

Comment:
In this spec + implementation, we explicitly do not want to process fallback videos in production. Fallback videos only ever need to be processed once, which will usually take place when the content is initially added by the author in the levelbuilder environment. Disabling the filter in production isolates concerns between authoring and delivery of this content, eliminating the video-processing dependencies from being installed in production and protecting the live site from any unintended bugs or issues related to timeouts or tying up worker threads you asked about in [another comment](https://github.com/code-dot-org/code-dot-org/pull/3260#discussion_r35794496).